docs/installation.rst

```diff
@@ -4,97 +4,99 @@
+Installing a package using miniforge
+------------------------------------
+To install ``sunpy``, launch a terminal (under a UNIX-like system) or miniforge Prompt (under Windows).
 
-To install the Miniconda Python distribution follow the instructions at
-`here <https://docs.conda.io/en/latest/miniconda.html>`__.
-Although Miniconda makes it simple to switch between Python versions, we recommend that new users install the latest Python 3.x version of Miniconda.
-
-The reason we choose Miniconda over Anaconda, is mainly due to the size as Anaconda comes with a full install of packages you probably do not need and this way you have more direct control over what has been installed into your Python virtual environment.
-Furthermore, you bypass the need for the conda resolver to sort out your root environment which should make conda faster to use.
-
-Installing sunpy using Miniconda
---------------------------------
+.. note::
 
-To install sunpy launch a system command prompt or the 'Anaconda Prompt' (under Windows).
-First configure conda for to add the `conda-forge channel <https://conda-forge.org/>`__::
+    We strongly recommend using a `Python virtual environment <https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/>`__ or a `conda virtual environment. <https://towardsdatascience.com/getting-started-with-python-environments-using-conda-32e9f2779307>`__
 
-    conda config --add channels conda-forge
-    conda config --set channel_priority strict
+You will want to activate your virtual environment then run:
 
-and now to install sunpy within the default conda virtual environment::
+.. code-block:: bash
 
     $ conda install sunpy
 
-This will install sunpy and every package it needs to function.
+This will install ``sunpy`` and each of its dependencies.
 
-.. note::
-    We strongly recommend using a `Python virtual environment <https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/>`__ or a `conda virtual environment. <https://towardsdatascience.com/getting-started-with-python-environments-using-conda-32e9f2779307>`__
+You can replace ``sunpy`` with other packages such as ``ablog`` or ``sunkit-instruments``.
+For the full range of packages available, `see the SunPy Affiliated Packages list <https://sunpy.org/project/affiliated>`__.
 
-Updating sunpy
---------------
+Updating a package
+------------------
+You can update to the latest version of any package by running:
 
-You can update to the latest version by running::
+.. code-block:: bash
 
-    conda update sunpy
-
-Installing sunpy on top of an existing scientific Python environment
---------------------------------------------------------------------
+    $ conda update <package_name>
 
+Installing on top of an existing scientific Python environment
```

Comment:
Should we just delete all of this and replace it with an "installing without conda" section? I feel like there's a lot of potential for user confusion if we don't keep it simple here.

Reply:
Yeah I will do that.

Reply:
👍